release-notes/v4-tucker/4.6.19.md

```diff
@@ -0,0 +1,10 @@
+---
+title: 4.6.19
+---
+
+### HarperDB 4.6.19
```

Comment:
nit: only 4.6.18 seems to have added HarperDB to the version heading and these 4.6.* are following suit